### ISSUE TITLE
Can't overwrite default buffer size

### ISSUE THREAD

**Issue by sobolewskikamil** [ExpediaGroup/graphql-kotlin#899]
Hi,

We are using `graphql-kotlin-spring-client:4.0.0-alpha.3` to retrieve quite big response from server. We stumble across following error:
```org.springframework.core.io.buffer.DataBufferLimitException: Exceeded limit on max bytes to buffer : 262144```

We are aware that this is WebClient configuration concern, so we have configured WebClient in the following way in order to make the limit higher:
```
val webClientBuilder = WebClient.builder()
            .exchangeStrategies(
                ExchangeStrategies.builder()
                    .codecs { configurer ->
                        configurer
                            .defaultCodecs()
                            .maxInMemorySize(100 * 1024 * 1024)
                    }
                    .build())
```
Afterwards we create `GraphQLWebClient` in the following way: 
```
private val client = GraphQLWebClient(url, objectMapper, webClientBuilder)
```

Unfortunately I doesn't work and somehow our configuration is lost. We suspect that it is caused by `init` block inside `GraphQLWebClient` class.
```
init {
        mapper.enable(DeserializationFeature.READ_UNKNOWN_ENUM_VALUES_USING_DEFAULT_VALUE)

        val strategies = ExchangeStrategies
            .builder()
            .codecs { configurer ->
                configurer.defaultCodecs().jackson2JsonEncoder(Jackson2JsonEncoder(mapper, MediaType.APPLICATION_JSON))
                configurer.defaultCodecs().jackson2JsonDecoder(Jackson2JsonDecoder(mapper, MediaType.APPLICATION_JSON))
            }.build()
        builder.exchangeStrategies(strategies)
    }
```

As you can see you are overwriting exchange strategies which causes configuration loss. Could you recommend some way to properly configure your client? Thanks :)

**Comment by dariuszkuc** [comment]
Nice catch! Indeed `init` will overwrite your configuration - I'll look into properly merging those configs

**Comment by sobolewskikamil** [comment]
@dariuszkuc Could you provide any ETA of the release with fix?

**Comment by dariuszkuc** [comment]
@sobolewskikamil I just released the fix in `4.0.0-alpha.6`
